src/main/java/se/sundsvall/incidentmapper/service/IncidentService.java

```diff
@@ -28,29 +32,66 @@
 import se.sundsvall.incidentmapper.integration.jira.JiraClient;
 import se.sundsvall.incidentmapper.integration.pob.POBClient;
 
+import generated.se.sundsvall.pob.PobMemo;
+import generated.se.sundsvall.pob.PobPayload;
+
 @Service
 @Transactional
 public class IncidentService {
 
-	private static final Logger LOGGER = LoggerFactory.getLogger(IncidentService.class);
+	public static final String PROBLEM = "Problem";
 
-	private static final String LOG_MSG_CLEANING_DELETE_RANGE = "Removing all incidents with modified '{}' (or earlier) and with status matching '{}'.";
+	public static final String SCOPE_ALL = "all";
```

Comment:
Move all POB-mapping stuff to a PobMapper (and let this class handles the logic only)?